tests/lang/s01_basics/s00_atoms/t01_statement.nim

```diff
@@ -146,24 +231,29 @@ block noreturn_statement:
       value = 2
 
     try:
-      ## When exception is raised it propagates through all function talls (if
-      ## any) until it reaches topmost level (main module from which the first
+      ## When exception is raised it propagates through all function calls (if
+      ## any) until it reaches the topmost level (main module/path from which the first
       ## function was called) or a `try` statement. For particular details on
       ## how exceptions affect control flow see respective part of the
       ## specification. In this case `try` was needed so subsequent `doAssert`
       ## would be executed correctly.
       raiseInside()
 
     except:
+      # `try`, `except`, `finally`, will be explored in their respective section.
```

Comment:
Link concrete specification session

Reply:
Actually it is not written yet, so add `FIXME` for the time being

Reply:
tbd - specification session still not written afaik